### ISSUE TITLE
Invalid filename in the stat-whitelist

### ISSUE THREAD

**Issue by ukulekek** [rpminspect/rpminspect#87]
Hi!
When I run:
```
rpminspect samba-4.11.2-8.el8 samba-4.11.2-2.el8
```
I get:
```
*** Invalid filename in the stat-whitelist: -rwsr-x---
*** From this invalid line:
***     root	disk	/usr/lib64/amanda/application/amgtar
munmap_chunk(): invalid pointer
Aborted (core dumped)
```

**Comment by dcantrell** [comment]
Ah yes.  Definitely need to work on this one.  Thanks for the reproducer.  I will be able to get to this next week.  rpminspect's ancestor has taken the majority of my time this week.

**Comment by dcantrell** [comment]
This one is strange.  I am unable to reproduce it locally.

Also, unrelated, but are you specifying the builds on the command in that order deliberately?  rpminspect expects the order OLD_BUILD NEW_BUILD, so I would expect the command to be "rpminspect samba-4.11.2-2.el8 samba-4.11.2-8.el8".  If you are doing the ordering on purpose, disregard the question.

**Comment by dcantrell** [comment]
@ukulekek are you still seeing this problem?  Do you have a more current reproducer?

**Comment by dcantrell** [comment]
@ukulekek I need more information on how you are running rpminspect here.  I have tried multiple ways to trigger this failure and can't do it.

**Comment by dcantrell** [comment]
Fixed in 238dd533b898170722af2514f23ac0e3cbf43e14
